src/modules/auth/views/activation-view.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="auth-view-container">
+    <AuthLoader
+      :is-visible="sendActivationTokenMutation.isPending.value"
+    />
+    <div
+      v-if="sendActivationTokenMutation.isError.value"
+      class="auth-view-container--error"
+    >
+      <v-icon>mdi-alert-circle</v-icon>
+      {{ errorMessage }}
+    </div>
+    <div
+      v-if="sendActivationTokenMutation.isSuccess.value"
+      class="auth-view-container--success"
+    >
+      <header class="d-flex justify-center align-center">
+        <v-icon :size="28">mdi-check-circle</v-icon>
+        <h3>{{ $t('Your account is active') }}</h3>
+      </header>
+
+      <p class="text text-center mt-5 mb-3">
+        {{
+          $t(
+            'You will be redirected to new password form in {secondsLeftToRedirect} seconds',
+            { secondsLeftToRedirect: secondsLeftToRedirect },
+          )
+        }}
```

Comment:
To jest dość ciekawe, obserwuję rozwój tych translacji od samego początku i zastanawiałem się czy to podejśćie jest odpowiednie - chyba że to po prostu luźne podejście na potrzeby kodu.

Zwykle w projektach komercyjnych tworzy się pary klucz: wartośc dla translacji, ale klucz nie jest w żaden sposób zwykłym zdaniem, czy odzwierciedleniem 1:1 wartości.

Zwykle robi się np. `redirectConfirmationMessage: You will be redirected to new password form in {secondsLeftToRedirect} seconds`.

Wtedy mamy krótszy klucz w markupach no i wydaje mi się że jest to ciut prostsze do utrzymania niż pełnoprawne zdanie. 

O ile poprzednie przykłady były jeszcze okej, to tutaj mamy w kluczu te brackety `{ }` co już wygląda dziwnie, dlatego poddaje to pod dyskusje.

Dodatkowo ponizej mozna uzyc shorthanda `{ secondsLeftToRedirect }`